_posts/2018-05-25-ecs-calculator.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: ECS-calculator
+banner: /assets/posts/2018-05-25-ecs-calculator/ecs-banner.png
+author:
+  - veranikaisakova
+  - hadoan
+---
+
+In the cloud, containers provide a containerised environment enabling your code to be built, shipped and run anywhere. This can be simply done by just running your code without the setting up of your operating system. Coming with such benefit, AWS Elastic Container Services (ECS) is a a highly scalable, fast, container management service that makes easy to run your containerised code and applications across a managed cluster of EC2 instances. You will need to optimally scale and fit ECS containers in an EC2 instance for the sake of efficient usage of your resources on cloud. This means that the CPU and memory capacity of the EC2 instance should be used up to be distributed over the possible maximum number of containers that can fit in the EC2 instance. Therefore, the EC2 Calculator project is developed to provide a quick access to data of the CPU and memory reservation for each EC2 instance type and their values divided into containers.
```

Comment:
You mention the EC2 Calculator project here. But I don't see a link to it, where can I find it?

Reply:
Yes, we should include a link to the project. Should we include both links to our end product  and code (https://github.com/sentiampc/aws-ecs-resource-calculator) ?